embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataSourceModule.kt

```diff
@@ -56,6 +58,7 @@ internal class DataSourceModuleImpl(
     systemServiceModule: SystemServiceModule,
     androidServicesModule: AndroidServicesModule,
     workerThreadModule: WorkerThreadModule,
+    anrModule: AnrModule
```

Comment:
Doesn't require a provider instead of the instance? Be careful with initialization.